tests/test_fractional_octaves.py

```diff
@@ -89,18 +90,31 @@ def test_fract_oct_filter_iec():
     impulse = pyfar.signals.impulse(n_samples, sampling_rate=sr)
 
     f_obj = filter.fractional_octave_bands(
-        None, 3, sampling_rate=sr, order=order)
+        None, 3, sampling_rate=sr)
     assert isinstance(f_obj, FilterSOS)
 
-    sig = filter.fractional_octave_bands(impulse, 3, order=order)
+    sig = filter.fractional_octave_bands(impulse, 3)
     assert isinstance(sig, Signal)
 
     ir_actual = filter.fractional_octave_bands(
-        impulse, 1, frequency_range=(1e3, 4e3), order=order)
+        impulse, 1, frequency_range=(1e3, 4e3))
```

Comment:
What about testing this also for very low frequencies?

Reply:
we could parametrize maybe 2 or 3 frequency ranges